Sources/BSON/Bool+BSONValue.swift

```diff
@@ -0,0 +1,21 @@
+import NIO
+
+extension Bool: BSONValue {
+    static var bsonType: BSONType { .bool }
```

Comment:
these need explicit acl's also. ditto for other `BSONValues` in this PR. It seems swiftlint doesn't properly check for `explicit_acl` in extensions unfortunately.